targets/XPlatCppSdk/source/XPlatCppSdk.sln.ejs

```diff
@@ -14,8 +14,6 @@ Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "cppWindowsTestApp", "testap
 		{92F6EC5F-B66C-4572-A74D-79329AFF3129} = {92F6EC5F-B66C-4572-A74D-79329AFF3129}
 	EndProjectSection
 EndProject
-Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "WindowsNugetTestApp", "testapps\cppWindowsTestApp\WindowsNugetTestApp.vcxproj", "{BE258938-E8B1-4DCA-8408-DCFD973F284F}"
```

Comment:
>Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "WindowsNugetTestApp", "testapps\cppWindowsTestApp\WindowsNugetTestApp.vcxproj" [](start = 0, length = 131)

Should we also delete all content under [https://github.com/PlayFab/SDKGenerator/tree/master/targets/XPlatCppSdk/source/testapps/cppWindowsTestApp](https://github.com/PlayFab/SDKGenerator/tree/master/targets/XPlatCppSdk/source/testapps/cppWindowsTestApp)
?